components/content/carousel/NetworkImage.tsx

```diff
@@ -43,24 +49,16 @@ export const NetworkImage: React.FC<NetworkImageProps> = ({uri, width, height, o
 
   return (
     <Center width={width} height={height}>
-      {state === 'loading' && <ActivityIndicator style={{height: Math.min(32, height / 2)}} />}
-      {state === 'error' && (
+      {status === 'loading' && <ActivityIndicator style={{height: Math.min(32, height / 2)}} />}
+      {status === 'error' && (
         <VStack space={8} alignItems="center">
           <FontAwesome5 name="exclamation-triangle" size={Math.min(32, height / 2)} color={COLORS['warning.700']} />
           <Body>Media failed to load.</Body>
         </VStack>
       )}
-      {(state === 'ready' || state === 'loading') && (
-        <TouchableOpacity activeOpacity={0.8} onPress={() => onPress(index)} disabled={state !== 'ready'}>
-          <Image
-            source={{uri}}
-            onLoad={() => {
-              setState('ready');
-              onStateChange('ready');
-            }}
-            onError={_e => setState('error')}
-            {...croppedThumbnailProps}
-          />
+      {(status === 'success' || status === 'loading') && (
+        <TouchableOpacity activeOpacity={0.8} onPress={() => onPress(index)} disabled={status !== 'success'}>
+          <Image source={{uri: cachedUri, cache: 'force-cache'}} {...croppedThumbnailProps} />
```

Comment:
`force-cache` is iOS only. it's also reading from the HTTP cache, not from your local filesystem cache that you wrote to in useCachedImageURI. I think this is probably working when you're testing, but maybe not as you intend?

Reply:
Oops, this is a hold-over from a previous attempt, we can remove these.